### DISCUSSION TITLE
WirelessHost  error while sending a TSN stream

### DISCUSSION THREAD

**Discussion by Abh4git** [inet-framework/inet#954] [General]
Scenaio:  I am trying to use a WirelessHost to send a TSN Stream (UDP Video, Best Effort) to AccessPoint. Ideally this should get QTag added. WirelessHost has bridging as well enabled with InterfaceRelay set to default("") (like in TSNDevice). 

Environment: I am using OMNeT++ Integrated Development Environment , Version: 7.0.0pre2, Build id: 230827-e42d35366e
and INET 4.5.2 

Error: I keep getting at MessageDispatcher - handlePacket - Unknown Interface : InterfaceId 101 
It for some reason does not identify the wlan0 interface. 
Also Moment i remove/comment bridging out it is able to send but does not add QTag to it . I want Qbv enabled to have TSN working. As I noticed making it a stream is key to next steps.I checked Bridging to see if anything is ethernet specific, but couldn't find. 

Please do let me know if you need more details. Thank you for any pointers.
Any thoughts why this is happening? Or is there a workaround?.

**Comment by rhornig** [comment]
TSN is a wired standard (at least for now). There are work in progress to integrate with 5G, but the TSN implementation in INET is strictly wired.

**Reply by Abh4git** [reply]
Thank you for clarifying!
